changes/11927-vuln-false-positive-icloud

```diff
@@ -0,0 +1,2 @@
+- Added CPEMatchingRule type, used for correcting false positives caused by incorrect entries in the
+NVD dataset.
```

Comment:
@juan-fdz-hawa Shouldn't this be a message for users?
Maybe something like:

**Fixed iCloud false positives** ?

Reply:
Thanks - I'll add a new entry to the changes file with that.

Reply:
Fixed in https://github.com/fleetdm/fleet/pull/12551/commits/efaf93dbc46664fce508d5abde02af145ff445c9